core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/PlayerConstants.kt

```diff
@@ -3,7 +3,7 @@ package com.pierfrancescosoffritti.androidyoutubeplayer.core.player
 class PlayerConstants {
 
     enum class PlayerState {
-        UNKNOWN, UNSTARTED, ENDED, PLAYING, PAUSED, BUFFERING, VIDEO_CUED
+        UNKNOWN, UNSTARTED, ENDED, PLAYING, PAUSED, BUFFERING, VIDEO_CUED, SHOW_CUSTOM_VIEW, HIDE_CUSTOM_VIEW
```

Comment:
These two constants are not related to the state of the player, please move them somewhere else

Reply:
Changes are made as per suggestion

1. Remove `SHOW_CUSTOM_VIEW` and `HIDE_CUSTOM_VIEW` from PlayerState
2. Update `SimpleFullscreenExampleActivity` code accordingly.